store/staffs.js

```diff
@@ -0,0 +1,65 @@
+import { make } from 'vuex-pathify'
+
+export const state = () => ({
+  staff: {
+    summary: {
+      contactLogs: {
+        needUpdatingContact: {}
+      }
+    }
+  },
+  count: 0
+})
+
+export const mutations = {
+  ...make.mutations(state)
+}
+
+export const getters = {
+  ...make.getters(state),
+  getTotalItemCount: (_) => {
+    let count = 0
+    const needUpdatingContact = _.staff.summary.contactLogs.needUpdatingContact
+    for (const key in needUpdatingContact) {
+      const value = needUpdatingContact[key]
+      if (value && typeof value === 'number') count += value
+    }
+    return count
+  },
```

Comment:
1. Không sử dụng for, thay vào đó sử dụng các function có sẵn của array.
2. Có thể cải tiến đoạn code này ví dụ:
```
getTotalItemCount: (_) => {
    return Object.values(
      _.staff.summary.contactLogs.needUpdatingContact
    ).reduce((totalCount, value) => {
      if (Number.isFinite(value)) {
        return (totalCount += value)
      }
    }, 0)
  }
```